gpt_index/data_structs/data_structs.py

```diff
@@ -56,6 +56,8 @@ def __post_init__(self) -> None:
     # extra node info
     node_info: Optional[Dict[str, Any]] = None
 
+    image: Optional[str] = None
```

Comment:
add a comment that the image is base64 encoded? i was confused why this was a `str` at first